component/azstorage/utils_test.go

```diff
@@ -75,6 +76,45 @@ type contentTypeVal struct {
 	result string
 }
 
+func (s *utilsTestSuite) TestPrefixPathRemoval() {
+	assert := assert.New(s.T())
+
+	type PrefixPath struct {
+		prefix string
+		path   string
+		result string
+	}
+
+	var inputs = []PrefixPath{
+		{prefix: "", path: "abc.txt", result: "abc.txt"},
+		{prefix: "", path: "ABC", result: "ABC"},
+		{prefix: "", path: "ABC/DEF.txt", result: "ABC/DEF.txt"},
+		{prefix: "", path: "ABC/DEF/1.txt", result: "ABC/DEF/1.txt"},
+
+		{prefix: "ABC", path: "ABC/DEF/1.txt", result: "DEF/1.txt"},
```

Comment:
can we ever expect a path with / at the end? is that something this method should support?